Makefile.e2e

```diff
@@ -168,12 +168,45 @@ test-e2e-go-ephemeral: __push-local-image
 	# The .sock volume allows you to connect to the Docker daemon of the host.
 	# Part of the docker-in-docker pattern.
 	@docker run -v /var/run/docker.sock:/var/run/docker.sock --network="host" prow-image make __test-e2e-go-nobuild-prow
+# This target runs all the e2e tests with the mono-repo mode.
 test-e2e-go-ephemeral-mono-repo: __push-local-image
 	kind delete clusters --all
 	./scripts/e2e-go-ephemeral.sh --timeout 60m --parallel 18 --image-tag=$(LATEST_IMAGE_TAG)
+# This target runs the first group of e2e tests with the mono-repo mode.
+test-e2e-go-ephemeral-mono-repo-group1: __push-local-image
```

Comment:
For consistency, can we rename it to `test-e2e-kind-mono-repo-test-group1`?

We're going to port all remaining bats test to go, so `go` is not needed in the target name.

`kind` indicates it is an ephemeral cluster.